platform/lego_nxt/cmds/bt_control/bt_control.c

```diff
@@ -16,7 +16,8 @@
 
 EMBOX_CMD(bt_main);
 
-static struct net_node *add_mod(const char *str_id, struct pnet_graph *graph, struct net_node *prev) {
+static struct net_node * add_mod(const char *str_id, struct pnet_graph *graph,
+	struct net_node *prev) {
```

Comment:
Unterminated list in parentheses should continue with 2 indentations on next line. This is not right, while fix below on line 50 is good (2 indentations to continued parameter list)